.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,52 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
+<summary><strong>Guidelines</strong></summary>
+Thank you for making a feature suggestion! :heart:
 
-<!-- Please fill these sections with the relevant information: -->
+**Please fill out all the sections below with the relevant information.**
 
-# A detailed description of the proposed feature
+We will review your suggestion and let you know our decision as soon as we can.
+Note, that we will close this issue if neither you nor we have the time to work on it.
 
-<!-- replace me -->
+Homebrew is a non-profit project run entirely by unpaid volunteers.
+We, therefore, reserve the right to close your issue without comment if you
+delete, do not read, or do not fill out the issue checklist below and provide
+_**all** information requested_. Note, that if you repeatedly fail to fill out
+the issue template, we will be forced to block you from ever submitting issues
+to Homebrew.
```

Comment:
```suggestion
the issue template, we will block you from submitting issues to Homebrew.
```

Reply:
I'm not sure about this...  `will be forced to` adds a hint of politeness to this rather strong message. It doesn't make the message any weaker and informs about our "great power". It also conveys the idea that we don't want to do that, which, I believe, is true.

Reply:
No one's really forcing us to do anything. It sounds unnecessarily wordy to me.

Reply:
How about:

```suggestion
the issue template, we may block you from ever submitting issues to Homebrew.
```

To me, it feels a little nicer than a plain old "we will block you" but is still right to the point

Reply:
> a plain old "we will block you"

We will we will _block you_ 👏
🕺 🕺 👏

Reply:
This should be the first line of the template and, whatever is done with the wording, the issue template should be made consistently here and in homebrew-core (which can be a follow-up PR after this is merged).